packages/xrpl/test/integration/requests/ledger.test.ts

```diff
@@ -55,6 +55,8 @@ describe('ledger', function () {
       assert.typeOf(ledgerResponse.result.ledger_index, 'number')
 
       const ledger = ledgerResponse.result.ledger as Ledger & {
+        // Add index signature to request to iterate through keys and asset types
+        [index: string]: unknown
```

Comment:
Similar to my comments in other test files, I don't see the need for this additional kv pair